src/ui/components/shared/Onboarding/BubbleBackground.module.css

```diff
@@ -0,0 +1,11 @@
+.container {
+  pointer-events: none;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  background: linear-gradient(to bottom, #081120 0%, #152746 50%, black 100%);
+
+  color: #d7d7db;
```

Comment:
⚠️ Hard-coded CSS colors; should be using theme values, no?

Reply:
In this particular case, we're forcing a single look and feel (that ignores theme) so I think this is appropriate.

But I'm open to reconsidering; I could make a single-use css variable for this one case to make sure I'm using a variable.  But that seemed odd. Thoughts?

Reply:
> In this particular case, we're forcing a single look and feel 

I see!

Even in this case, I think it's generally easier to manage complexities to create new theme variables (that just happen to have the same color values– at least for now)

Reply:
Sounds good! I'm going to merge this to get it live and follow-up on the CSS standardisation because I think there are other things to hunt down too: https://linear.app/replay/issue/DES-674/standardise-our-onboarding-css-with-variables-not-hard-coded-hex

Thank you!